cli/index.ts

```diff
@@ -66,11 +66,17 @@ const includeDepsOption: INamedOption<yargs.Options> = {
   }
 };
 
-const schemaSuffixOption: INamedOption<yargs.Options> = {
+const schemaSuffixOverrideOption: INamedOption<yargs.Options> = {
   name: "schema-suffix",
   option: {
-    describe: "A suffix to add to any output schema names.",
-    
+    describe: "A suffix to be appended to output schema names."
+  },
+  check: (argv: yargs.Arguments) => {
+    if (argv.schemaSuffix && !/^[a-zA-Z_0-9]+$/.test(argv.schemaSuffix)) {
+      throw new Error(
+        "--schema-suffix must containly only lowercase, uppercase, underscore and number characters."
```

Comment:
"contain"

suggest:

--schema-suffix should contain only alphanumeric characters and/or underscores.